src/prescriptions/prescription-tab-lists.component.tsx

```diff
@@ -36,41 +12,15 @@ enum TabTypes {
   ALL,
 }
 
-const labelMap = [
-  "All prescriptions",
-  "Active prescriptions",
-  "Complete",
-  "Returned",
-  "Cancelled",
-];
-
-const columns = [
-  { header: "Created", key: "created" },
-  { header: "Patient name", key: "patientName" },
-  { header: "Prescriber", key: "prescriber" },
-  { header: "Drugs", key: "drugs" },
-  { header: "Last dispenser", key: "lastDispenser" },
-  { header: "Status", key: "status" },
+const tabs = [
+  { label: "activePrescriptions", status: "ACTIVE" },
+  { label: "allPrescriptions", status: "" },
```

Comment:
Based on discussion with our BAs, I simplified this to two tabs, "active" and "all", since they were the most straightforward to define and implement.  I also switched "active" to be the default tab.  We can continue to discuss what other tabs might be MVP.

Each tab now contains a label and a status; the status is passed to the underlying FHIR call, in order to request only orders with a specific status.